test_files/Skip_checks_test.cif

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
I don't think we actually need a new test file. We can also test this loading an existing CIF test file using `open('some_file.cif')` and then modify the occupancies in that string before passing it to `CifParser.from_string`.

Reply:
I hope it’s okay to do this, but I’m going to address all the remaining requested changes in this reply before I push the commit. 

1. I’ve added hints to the _get_structures function. Did you want me to add hints to the get_structures function as well or anywhere else for that matter?
2. I renamed skip_checks to skip_occu_checks everywhere I use it and it is defaulted to False. 
3. I am not sure what you mean about your white space comment. Can you clarify?
4. Skip_checks_test.cif has been removed. It now tests site_type_symbol_test.cif and changes the Te occupancy to 1.5000.

Is there anything else I am missing here before I hit go on my next commit?